src/index.js

```diff
@@ -1,24 +1,27 @@
 const util = require('util');
 const fse = require('fs-extra');
-const glob = require('glob');
+const {glob} = require('glob');
 const core = require('@actions/core');
 const exec = require('@actions/exec');
 const github = require('@actions/github');
 const changelogParser = require('changelog-parser');
 const npmFetch = require('npm-registry-fetch');
 
-const globPromise = util.promisify(glob);
-
-const fetchNPMVersions = (packageName, token) => {
-    return npmFetch.json(
+/**
+ * @param {string} packageName 
+ * @param {string} token
+ */
+const fetchNPMVersions = async (packageName, token) => {
```

Comment:
Return value is missing

Reply:
It's in the next line no?

Reply:
Type wise I  mean `@returns`

Reply:
I left it out because the inferred value was correct, or you are with ThePrimeagen on this and want all returns to be manually typed?

<img width="729" alt="Screenshot 2024-02-15 at 14 00 55" src="https://github.com/zattoo/publish/assets/1938043/18911b21-b500-411e-9d98-a9001176ab9f">

Reply:
Na it's fine